patch-package
new file mode 100644
Binary files /dev/null and b/node_modules/cypress/dist/Cypress.app/Contents/Resources/app/packages/extension/theme/Cached Theme.pak differ
--- a/node_modules/cypress/dist/info.json
+++ b/node_modules/cypress/dist/info.json
@@ -1,3 +1,4 @@
 {
-  "version": "2.1.0"
+  "version": "2.1.0",
+  "verifiedVersion": "2.1.0"
 }
--- a/node_modules/cypress/types/index.d.ts
+++ b/node_modules/cypress/types/index.d.ts
@@ -67,13 +67,13 @@ declare namespace Cypress {
      * @example
      *    Cypress.Blob.method()
      */
-    Blob: BlobUtil.BlobUtilStatic
+    Blob: any
     /**
      * Cypress automatically includes minimatch and exposes it as Cypress.minimatch.
      *
      * @see https://on.cypress.io/minimatch
      */
-    minimatch: Mimimatch.MimimatchStatic
+    minimatch: any
     /**
      * Cypress automatically includes moment.js and exposes it as Cypress.moment.
      *
@@ -91,7 +91,7 @@ declare namespace Cypress {
      * @example
      *   new Cypress.Promise((resolve, reject) => { ... })
      */
-    Promise: Bluebird.BluebirdStatic
+    Promise: any
     /**
      * Cypress version string. i.e. "1.1.2"
      * @see https://on.cypress.io/version
